easybuild/easyconfigs/a/AFNI/AFNI-linux_openmp_64-goolf-1.5.14.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Bart Verleye
+# Center for eResearch, Auckland
+
+
+name = 'AFNI'
+version = 'linux_openmp_64'
+# see VersionDate in caps3 output
+
+easyblock = 'PackedBinary'
+
+homepage = 'http://afni.nimh.nih.gov'
+description = """Free software for analysis and display of FMRI data """
+
+toolchain = {'name': 'goolf', 'version': '1.5.14'}
+
+source_urls = [('http://afni.nimh.nih.gov/pub/dist/tgz/')]
+sources = ['%(version)s.tgz']
```

Comment:
and probably a versionsuffix like `-20141023` too (the datestamp on the file on the source URL), to allow for future updates